collateral_pools/collateral_pool_1/risk/risk_matrix.toml

```diff
@@ -0,0 +1,77 @@
+# [invoke.cp_1rusd_create_risk_matrix]
+# target = ["CoreProxy"]
+# abi = "<%= JSON.stringify(contracts.CoreProxy.abi.concat({'type':'event','name':'RiskMatrixCreation','inputs':[{'name':'collateralPoolId','type':'uint128','indexed':true,'internalType':'uint128'},{'name':'riskBlockId','type':'uint256','indexed':true,'internalType':'uint256'},{'name':'riskMatrix','type':'int64[][]','indexed':false,'internalType':'SD1x18[][]'},{'name':'blockTimestamp','type':'uint256','indexed':false,'internalType':'uint256'}],'anonymous':false})) %>"
+# fromCall.func = "getCollateralPoolOwner"
+# fromCall.args = ["<%= settings.cp1RusdId %>"]
+# func = "createRiskMatrix"
+# var.cp1RusdRiskBlockId.event = "RiskMatrixCreation"
+# var.cp1RusdRiskBlockId.arg = 1
+# args = [
+#   "<%= settings.cp1RusdId %>",
+#   [
+#     ["<%= parseEther(settings.cp1Rusd_riskMatrix00Unscaled) %>", "<%= parseEther(settings.cp1Rusd_riskMatrix01Unscaled) %>"],
+#     ["<%= parseEther(settings.cp1Rusd_riskMatrix10Unscaled) %>", "<%= parseEther(settings.cp1Rusd_riskMatrix11Unscaled) %>"],
+#   ]
+# ]
+# depends = [
+#   "var.cp_1rusd_vars"
+# ]
+
+[invoke.cp_1rusd_create_risk_matrix_market_1eth]
+target = ["CoreProxy"]
+abi = "<%= JSON.stringify(contracts.CoreProxy.abi.concat({'type':'event','name':'RiskMatrixCreation','inputs':[{'name':'collateralPoolId','type':'uint128','indexed':true,'internalType':'uint128'},{'name':'riskBlockId','type':'uint256','indexed':true,'internalType':'uint256'},{'name':'riskMatrix','type':'int64[][]','indexed':false,'internalType':'SD1x18[][]'},{'name':'blockTimestamp','type':'uint256','indexed':false,'internalType':'uint256'}],'anonymous':false})) %>"
+fromCall.func = "getCollateralPoolOwner"
+fromCall.args = ["<%= settings.cp1RusdId %>"]
+func = "createRiskMatrix"
+var.cp1RusdMarket1EthRiskBlockId.event = "RiskMatrixCreation"
+var.cp1RusdMarket1EthRiskBlockId.arg = 1
+args = [
+  "<%= settings.cp1RusdId %>",
+  [["<%= parseEther(settings.cp1Rusd_market1Eth_riskMatrix00Unscaled) %>"]]
+]
+depends = [
+  "var.cp_1rusd_vars"
+]
+
+[invoke.cp_1rusd_create_risk_matrix_market_2btc]
+target = ["CoreProxy"]
+abi = "<%= JSON.stringify(contracts.CoreProxy.abi.concat({'type':'event','name':'RiskMatrixCreation','inputs':[{'name':'collateralPoolId','type':'uint128','indexed':true,'internalType':'uint128'},{'name':'riskBlockId','type':'uint256','indexed':true,'internalType':'uint256'},{'name':'riskMatrix','type':'int64[][]','indexed':false,'internalType':'SD1x18[][]'},{'name':'blockTimestamp','type':'uint256','indexed':false,'internalType':'uint256'}],'anonymous':false})) %>"
+fromCall.func = "getCollateralPoolOwner"
+fromCall.args = ["<%= settings.cp1RusdId %>"]
+func = "createRiskMatrix"
+var.cp1RusdMarket2BtcRiskBlockId.event = "RiskMatrixCreation"
+var.cp1RusdMarket2BtcRiskBlockId.arg = 1
+args = [
```

Comment:
why do we want two risk blocks of 1x1 rather than one of 2x2 with zero in corner cases to see it how it goes in prod?